pkg/lightning/backend/local/region_job.go

```diff
@@ -802,12 +802,17 @@ func (local *Backend) doIngest(ctx context.Context, j *regionJob) (*sst.IngestRe
 
 // UpdateWriteSpeedLimit updates the write limiter of the backend.
 func (local *Backend) UpdateWriteSpeedLimit(limit int) {
-	local.writeLimiter.UpdateLimit(limit)
+	if local.writeLimiter != nil {
+		local.writeLimiter.UpdateLimit(limit)
+	}
 }
 
 // GetWriteSpeedLimit returns the speed of the write limiter.
 func (local *Backend) GetWriteSpeedLimit() int {
-	return local.writeLimiter.Limit()
+	if local.writeLimiter != nil {
```

Comment:
no need nil check

Reply:
In MockBackendCtx, this field is nil and fail the test with a certain probability.

Reply:
if so, then init it

and I see we use a `MockBackendCtx` in real tikv test, why not use the real one directly?

Reply:
`MockBackendCtx` is also used in unit tests in `ddl/ingest`